frontends/web/.eslintrc.js

```diff
@@ -0,0 +1,214 @@
+    "no-unneeded-ternary": 2,
+    "no-unused-vars": [2, { args: "none", varsIgnorePattern: "^h$" }],
+    "no-useless-call": 2,
+    "no-useless-computed-key": 2,
+    "no-useless-concat": 2,
+    // "no-useless-constructor": 2,
+    "no-useless-constructor": "off",
+    "no-useless-escape": 2,
+    "no-useless-rename": 2,
+    "no-var": 2,
+    "no-with": 2,
+    "object-shorthand": 2,
+    "prefer-arrow-callback": 2,
+    "prefer-rest-params": 2,
+    "prefer-spread": 2,
+    "prefer-template": 0, // we can enable this when they add an option to allow concat for complex variables
+    // "quote-props": [2, "as-needed"],
+    "quote-props": "off",
+    radix: 2,
+    "rest-spread-spacing": 2,
+    semi: 2,
+    "space-before-function-paren": [0, "always"], // for fmt
+    "space-in-parens": [0, "never"],
+    strict: [2, "never"],
+    "unicode-bom": 2,
+	  "valid-jsdoc": [0, { requireReturn: false }], // no way to turn this off for pure functional component
+    
+    // Shift
+    "no-alert": "off",
+    "arrow-body-style": [0, "as-needed"],
+    // brace-style": [1, "1tbs"],
+    "brace-style": "off",
+    "comma-dangle": 0,
+    // "indent": ["warn", 4, {
+    //   "SwitchCase": 1
+    // }],
+    "indent": "off",
+    "quotes": ["warn", "single", {
+      "avoidEscape": true,
+      "allowTemplateLiterals": true
+    }],
+    "quote-props": ["warn", "as-needed"],
+    "react/self-closing-comp": 0,
+    "react/jsx-curly-spacing": 0,
+    // "react/jsx-indent-props": ["warn", 4],
+    "react/jsx-indent-props": "off",
+    "react/sort-comp": 0,
+    "react/jsx-boolean-value": 0,
+    "react/jsx-closing-bracket-location": 0,
+    // "react/jsx-max-props-per-line": ["warn", { "maximum": 5, "when": "always" }],
+    "react/jsx-max-props-per-line": "off",
+    // "react/jsx-tag-spacing": ["warn"],
+    "react/jsx-tag-spacing": "off",
+    "react/jsx-no-bind": "off",
+    // "react/jsx-key": ["error"],
+    "react/jsx-key": "off",
+
+    "object-curly-spacing": "off",
+
+    // TODO: enable again
+    "no-undef": "off",
+    "react/no-unknown-property": "off",
+    "react/no-children-prop": "off",
+
+    // TODO: enable again
+    "@typescript-eslint/no-unsafe-call": "off",
+    "@typescript-eslint/no-unsafe-return": "off",
+    "@typescript-eslint/no-unsafe-assignment": "off",
+    "@typescript-eslint/restrict-plus-operands": "off",
+    "@typescript-eslint/explicit-module-boundary-types": "off",
+    "@typescript-eslint/no-unsafe-member-access": "off",
+    "@typescript-eslint/no-floating-promises": "off",
+    "@typescript-eslint/ban-types": "off",
+    "@typescript-eslint/ban-ts-comment": "off",
+    "@typescript-eslint/no-explicit-any": "off",
+    "@typescript-eslint/unbound-method": "off",
+    "@typescript-eslint/no-non-null-assertion": "off",
+    "@typescript-eslint/no-unnecessary-type-assertion": "off",
+    "@typescript-eslint/restrict-template-expressions": "off",
+    "@typescript-eslint/no-empty-interface": "off",
```

Comment:
i hope we can keep this one disabled :) i like `interface Props {}` better, so it is named, than inlining `()` everywhere.

Reply:
👍  moved it from TODO to the Shift section